test/Shared/SyncPoint.cs

```diff
@@ -20,7 +20,7 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Grpc.AspNetCore.Server.Tests
+namespace Grpc.Tests
```

Comment:
nit: use a slightly less general namespace than Grpc.Tests?

Reply:
Changed to Grpc.Tests.Shared.